### PR TITLE
Fix minor typo

### PR DESCRIPTION
**Pull request by yurchor**
Thanks in advance for reviewing.

### CODE REVIEW

File: weblate_web/templates/snippets/service.html [hunk 1]
```diff
@@ -93,7 +93,7 @@
                         <a class="link" href="{{ service.last_report.site_url }}">{{ service.last_report.site_title|default:"Weblate" }}</a>
                 {% else %}
                         <p class="text-muted">
-                            {% trans "You can use use support services on our webite. Or, for more comfort, use an activation token to connect your Weblate installation to the support portal. More in " %}
+                            {% trans "You can use support services on our webite. Or, for more comfort, use an activation token to connect your Weblate installation to the support portal. More in " %}
```
Comment (nijel) [inline weblate_web/templates/snippets/service.html:96]: There is also "webite" typo, cc @orangesunny

File: weblate_web/templates/snippets/service.html [hunk 2]
```diff
@@ -93,7 +93,7 @@
                         <a class="link" href="{{ service.last_report.site_url }}">{{ service.last_report.site_title|default:"Weblate" }}</a>
                 {% else %}
                         <p class="text-muted">
-                            {% trans "You can use use support services on our webite. Or, for more comfort, use an activation token to connect your Weblate installation to the support portal. More in " %}
+                            {% trans "You can use support services on our website. Or, for more comfort, use an activation token to connect your Weblate installation to the support portal. More in " %}
                             <a href="https://docs.weblate.org/en/latest/admin/support.html" class="link">{% trans "Documentation." %}</a>
```
Comment (nijel) [inline weblate_web/templates/snippets/service.html:97]: Avoid concatenating localized strings:

```suggestion
                            {% blocktrans with link_start='<a href="https://docs.weblate.org/en/latest/admin/support.html" class="link">' link_end='</a>' %}You can use support services on our website. Or, for more comfort, use an activation token to connect your Weblate installation to the support portal. More in {{ link_start }}documentation{{ link_end }}.{% endblocktrans %}
```

**Review by orangesunny — APPROVED** [review]
approve with small changes :)

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**